src/main/java/org/jenkinsci/plugins/mesos/MesosComputer.java

```diff
@@ -45,4 +47,19 @@ public void taskCompletedWithProblems(
   public String toString() {
     return String.format("%s (slave: %s)", getName(), getNode());
   }
+
+  @Override
+  public MesosSlave getNode() {
+    return super.getNode();
+  }
+
+  @Override
+  public HttpResponse doDoDelete() {
+    try {
+      getNode().terminate();
+    } catch (Exception e) {
+      logger.warn("Error killing " + getNode().getPodId());
```

Comment:
Please do not swallow exception and if so log the exception. We will figure out how to crash in [DCOS_OSS-5055](https://jira.mesosphere.com/browse/DCOS_OSS-5055) until then we should simply bubble up the exceptions to Jenkins and let it handle it.

Reply:
kk can only handle IOException from overriden method I am logging the InterruptedException for now